plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/semantics/SQLBackgroundParsingJob.java

```diff
@@ -356,6 +357,9 @@ public Interval runWithResult() {
             SQLParserContext parserContext = new SQLParserContext(this.editor.getDataSource(), this.editor.getSyntaxManager(), this.editor.getRuleManager(), this.document);
             List<SQLScriptElement> elements = SQLScriptParser.extractScriptQueries(parserContext, workOffset, workLength, false, false, false);
             if (elements.isEmpty()) {
+                if (DEBUG) {
+                    System.out.println("No script elements to parse in range " + workOffset + "+" + workLength);
```

Comment:
But why? We have loggers, use them instead

Reply:
I don't want to log it at all. I need it to debug if something goes wrong. When this feature become stable, it will be deleted.